microsoft-service-bus/2.2/modules/ROOT/pages/index.adoc

```diff
@@ -259,18 +282,18 @@ To enable the SSL checks, follow these steps to import the certificate:
 . Use the PowerShell cmdlet https://msdn.microsoft.com/library/azure/jj248762%28v=azure.10%29.aspx[Get-SBAutoGeneratedCA] to download the certificate locally in the machine running the Windows Service Bus. +
 For this example, the certificate file is exported to `%temp%\AutoGeneratedCA.cer`.
 . Running as Administrator, go to `%programfiles%\Java\jre7`. +
-you must be running as Administrator to perform a certificate import with `Keytool.exe`. 
-. Verify that the `bin\keytool.exe` tool and `lib\security\cacerts` exists. 
+you must be running as Administrator to perform a certificate import with `Keytool.exe`.
```

Comment:
```suggestion
You must be running as Administrator to perform a certificate import with `Keytool.exe`.
```

Reply:
I don't know why this was green because I didn't update it. I will fix the errors, though.